modules/unused-kubernetes-roles.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="unused-kubernetes-roles_{context}"]
+= Finding unused Kubernetes roles
+
+[role="_abstract"]
+Use the *Configuration Management* view to get more information about your Kubernetes roles and find unused roles.
+
+.Procedure
+. Navigate to the RHACS portal and click *Configuration Management* from the left-hand navigation menu.
+. Select *RBAC Visibility and Configuration* -> *Roles* from the *Configuration Management* view header.
+The *Roles* view displays a list of Kubernetes roles across your clusters, the permissions they grant, and where they are used.
+. Select a row (or one of the underlined links in a row) to view more details about the role.
+The information appears in a panel on the right.
+. To find roles not granted to anyone, select the *Users & Groups* column header.
+Then, while holding the kbd:[Shift] key, select the *Service Account* column header.
+The list gets sorted to show roles that are not granted to any users, groups, or service accounts.
```

Comment:
You could minimise this sentence as follows:

"The list shows the roles that are not granted to any users, groups, or service accounts."